src/IOCapture.jl

```diff
@@ -16,10 +16,19 @@ in the terminal. Returns an object with the following fields:
 
 The behaviour can be customized with the following keyword arguments:
 
-* `throwerrors`: if set to `true` (default), `iocapture` will rethrow any exceptions thrown by
-  `f`. If set to `false`, exceptions are also captured and the exception objects returned
-  via the `.value` field (with also `.error` and `.backtrace` set accordingly). If set to
-  `:interrupt`, only `InterruptException`s are rethrown.
+* `throwerrors`:
+
+  When set to `Any` (default), `iocapture` will rethrow any exceptions thrown
+  by evaluating `f`. Setting it to `true` has the same effect as `Any`.
+
+  Setting it to `Union{}` will capture all thrown exceptions and return them
+  via the `.value` field, and will also set `.error` and `.backtrace`
+  accordingly. Setting it to `false` also has this effect.
+
+  To throw on a subset of possible exceptions pass the exception type instead,
+  such as `InterruptException`. If multiple exception types may need to be
+  thrown then pass a `Union{...}` of the types. Passing `:interrupt` has the
+  same effect as using `InterruptException`.
```

Comment:
I would emphasize here that the `InterruptException` case is the one you usually want, e.g. by changing the order of the paragraphs.